packages/tldraw/src/lib/shapes/text/TextShapeUtil.tsx

```diff
@@ -329,7 +326,7 @@ function getTextSize(editor: Editor, props: TLTextShape['props']) {
 	}
 
 	return {
-		width: Math.max(minWidth, result.w),
+		width: manualWidth ?? Math.max(minWidth, result.w),
```

Comment:
nit: found it hard to follow the logic here safely (b/c autoSize has a different branch of logic)

suggestion: `autoSize ? Math.max(minWidth, result.w) : maybeManualWidth` to indicate intent better